Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -1946,24 +1948,37 @@ public String generateSessionToken(String str) {
     }
 
     @JavascriptInterface
-    public void renderBase64Image(String url, String id) {
-        String base64Image = getAPIResponse(url);
-        activity.runOnUiThread(() -> {
-            try {
-                if (!base64Image.equals("") && base64Image != null && id != null) {
-                    LinearLayout layout = activity.findViewById(Integer.parseInt(id));
-                    byte[] decodedString = Base64.decode(base64Image, Base64.DEFAULT);
-                    Bitmap decodedByte = BitmapFactory.decodeByteArray(decodedString, 0, decodedString.length);
-                    ImageView imageView = new ImageView(context);
-                    imageView.setImageBitmap(decodedByte);
-                    imageView.setScaleType(ImageView.ScaleType.FIT_CENTER);
-                    imageView.setAdjustViewBounds(true);
-                    imageView.setClipToOutline(true);
-                    layout.removeAllViews();
-                    layout.addView(imageView);
+    public void renderBase64Image(String url, String id, boolean fitCenter) {
```

Comment:
The new function is backwards compatible. The new case(the http check) that I have added doesn't actually effect because I am directly calling the `renderBase64ImageFile` function if there is no need of getting url response.

Reply:
This new function isn't breaking any old code. Do you still want me to add the old function.

Reply:
you added one extra parameter in function so it will break

Reply:
can you handle this behavior in js? also keep the function in case of reverting

Reply:
@Vignesh-772, previously it was only using fitcenter. So I have added `true` to all the occurences. So there will not be problem with that parameter.

Reply:
@witcher-shailesh, which behaviour are you referencing here?

Reply:
@Vignesh-772 can you switch to this branch and make this changes? Need to get this merged quickly